### ISSUE TITLE
Change url

### ISSUE THREAD

**Issue by naouss80** [rawmind0/alpine-kafka#12]
Please change url to https://archive.apache.org/dist/kafka/
As I am playing with pyspak there is only compatibilty with versions < 0.10
Almost noone hosts these versions but archive.apache.org

https://github.com/rawmind0/alpine-kafka/blob/355eeb7cf21fbb1486b67dcba0b29c82b2820c55/Dockerfile#L13

**Comment by rawmind0** [comment]
Excuse me, but don't undertand... This is just for build the docker...There are docker versions < 0.10 already released, rawmind0/alpine-kafka:0.9.0.1-2

**Comment by naouss80** [comment]
Perfect!
Makes sense.
Here is what I got for some reason
![image](https://user-images.githubusercontent.com/6564744/33534021-83bd6816-d86a-11e7-918e-1cf6a20a24eb.png)

In the meantime,
I tried building this myself by only changing the kafka version parameter and this could not download the kafka tgz files for versions <10 from the link you have in the code.
I suggested the url https://archive.apache.org/dist/kafka/ as this should contain all the kafka tgz versions.
I applied all these changes and it seems to work on my machine.
It also seems that the .sh files should be converted from dos to unix (see dos2unix)
